docs/other-tools/kubernetes/microk8s.md

```diff
@@ -8,122 +8,124 @@ We are using following setting for this purpose:
-```
+    ```sh
+    apt-get update && apt-get upgrade -y
+    ```
 
 - Install MicroK8s:
 
-```sh
-sudo snap install microk8s --classic
-```
+    ```sh
+    sudo snap install microk8s --classic
+    ```
 
 - Check the status while Kubernetes starts
 
-```sh
-microk8s status --wait-ready
-```
+    ```sh
+    microk8s status --wait-ready
+    ```
 
 - Turn on the services you want:
 
-```sh
-microk8s enable dns dashboard
-```
+    ```sh
+    microk8s enable dns dashboard
+    ```
 
-Try `microk8s enable --help` for a list of available services and optional features.
-`microk8s disable <name>` turns off a service. For example other useful services
-are: `microk8s enable registry istio storage`
+    Try `microk8s enable --help` for a list of available services and optional features.
+    `microk8s disable <name>` turns off a service. For example other useful services
+    are: `microk8s enable registry istio storage`
 
 - Start using Kubernetes
 
-```sh
-microk8s kubectl get all --all-namespaces
-```
+    ```sh
+    microk8s kubectl get all --all-namespaces
+    ```
 
-If you mainly use MicroK8s you can make our kubectl the default one on your command-line
-with `alias mkctl="microk8s kubectl"`. Since it is a standard upstream kubectl, you
-can also drive other Kubernetes clusters with it by pointing to the respective kubeconfig
-file via the `--kubeconfig` argument.
+    If you mainly use MicroK8s you can make our kubectl the default one on your
+    command-line with `alias mkctl="microk8s kubectl"`. Since it is a standard
+    upstream kubectl, you can also drive other Kubernetes clusters with it by
+    pointing to the respective kubeconfig file via the `--kubeconfig` argument.
 
 - Access the [Kubernetes dashboard](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/)
 UI:
 
-![Microk8s Dashboard Ports](images/microk8s_dashboard_ports.png)
+    ![Microk8s Dashboard Ports](images/microk8s_dashboard_ports.png)
 
-As we see above the kubernetes-dashboard service in the kube-system namespace has
-a ClusterIP of 10.152.183.73 and listens on TCP port 443. The ClusterIP is randomly
-assigned, so if you follow these steps on your host, make sure you check the IP
-adress you got.
+    As we see above the kubernetes-dashboard service in the kube-system namespace
+    has a ClusterIP of 10.152.183.73 and listens on TCP port 443. The ClusterIP
+    is randomly assigned, so if you follow these steps on your host, make sure
+    you check the IP adress you got.
 
-!!! note "Note"
-    Another way to access the default token to be used for the dashboard access
-    can be retrieved with:
-    ![How to retrive The kubernetes-dashboad Token](images/k8s-dasboard-retrive-token.png)
+    !!! note "Note"
+        Another way to access the default token to be used for the dashboard access
+        can be retrieved with:
+        ![How to retrive The kubernetes-dashboad Token](images/k8s-dasboard-retrive-token.png)
```

Comment:
Why is this an image instead of text that users can copy?

Reply:
markdown lint need to be disable by using #<!-- markdownlint-disable -->

Reply:
I replaced the image with code.